FEM/rf_bc_new.cpp

```diff
@@ -1190,14 +1190,14 @@ void CBoundaryConditionsGroup::Set(CRFProcess* pcs, int ShiftInNodeVector, const
 				m_polyline = GEOGetPLYByName(bc->geo_name);
 				// 08/2010 TF get the polyline data structure
 				GEOLIB::Polyline const* ply(static_cast<const GEOLIB::Polyline*>(bc->getGeoObj()));
-
 				if (m_polyline)
 				{
 					if (bc->getProcessDistributionType() == FiniteElement::CONSTANT
 					    || bc->getProcessDistributionType() == FiniteElement::SWITCH)
 					{
 						// 08/2010 TF
 						double msh_min_edge_length = m_msh->getMinEdgeLength();
+						if (m_polyline->epsilon == 0.0) m_polyline->epsilon = msh_min_edge_length;
```

Comment:
Just tested. ``if (m_polyline->epsilon == 0.0)`` does not work, it should be ```if (m_polyline->epsilon < DBL_MIN)``

Besides, instead of changing several lines in different files, you can put the following lines in file0.cpp, after line 260, where  ``if (!mesh_vec.empty())`` is ended.
```
    for (std::size_t i=0; i<polyline_vector.size(); i++)
	{
		if (std::fabs(polyline_vector[i]->epsilon) < DBL_MIN)
			polyline_vector[i]->epsilon = mesh_vec[0]->getMinEdgeLength();        
	}
```
I have tested the above change, and it works.

Reply:
Thank you Wenqing, I will follow your suggestions